src/transcript-plugin.tsx

```diff
@@ -410,8 +419,11 @@ export class TranscriptPlugin extends KalturaPlayer.core.BasePlugin {
     if (Math.max(this._transcriptPanel, this._transcriptIcon) > 0) {
       this.sidePanelsManager?.remove(this._transcriptPanel);
       this.upperBarManager!.remove(this._transcriptIcon);
+      // @ts-ignore
+      this.player.getService('AudioPluginsManager').remove(this._audioPlayerIconId);
```

Comment:
you can add a getter function for the service

Reply:
we need to remove the former not do the later